Point2DArr trajectory
CarState carState
float32 speed
float32 yawRate
